### PR TITLE
Version upgrades

### PR DESCRIPTION
**Pull request by dbu**
continue #58 

I did not adress these issues listed in #58

* Make sure configuration is loadable when defined as XML
* The XML schema file for the config is out-of-date with Configuration
* Translatable content. It might work; untested.

i suggest we just create issues for that and don't try to do that in this PR.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Tests/Resources/app/cache
 Tests/Resources/app/logs
+bin/
 composer.lock
-vendor
+vendor/
+.idea
```
Comment (dantleech) [inline .gitignore]: Did we not say not to include IDE specifc things in the ignore file?

**Comment by lsmith77** [comment]
+1 for merging as is and opening tickets for the other stuff

**Comment by dbu** [comment]
as we use the phpcr_document form type, this is blocked by https://github.com/doctrine/DoctrinePHPCRBundle/issues/193

**Comment by aitboudad** [comment]
you should rename rdf mappings file of `Post` to

`Symfony.Cmf.Bundle.BlogBundle.Doctrine.Phpcr.Post.xml`

**Comment by lsmith77** [comment]
@aitboudad thanks .. done

File: rsa [hunk 2]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (dbu) [inline rsa]: huh?

**Comment by dbu** [comment]
and we seem to have a conflict between routing auto and the routing auto bundle

File: .travis.yml [hunk 3]
```diff
@@ -1,30 +1,32 @@
...
-    - 5.5
+  - 5.4
+  - 5.5
+  - 5.6
+  - hhvm
 
 env:
-  - SYMFONY_VERSION=2.5.*
-  
+  - SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
```
Comment (WouterJ) [inline .travis.yml]: are we sure we removed all deprecated usages from this bundle?

File: .travis.yml [hunk 4]
```diff
@@ -1,30 +1,32 @@
...
     - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
+      env: SYMFONY_VERSION=2.6.*
     - php: 5.5
-      env: SYMFONY_VERSION=dev-master
-    
-
+      env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+    - php: 5.5
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
```
Comment (WouterJ) [inline .travis.yml]: missing allowed failures for:

* Symfony 2.8 (which should not set the deprecations helper to weak btw, so we can catch new deprecations in 2.8)
* Symfony 3.0
* PHP nightly

File: .travis.yml [hunk 5]
```diff
@@ -1,30 +1,32 @@
...
     - php: 5.5
       env: SYMFONY_VERSION=2.3.*
     - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
+      env: SYMFONY_VERSION=2.6.*
     - php: 5.5
-      env: SYMFONY_VERSION=dev-master
-    
-
+      env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
```
Comment (WouterJ) [inline .travis.yml]: this one is no longer needed as it's the default env

File: .travis.yml [hunk 6]
```diff
@@ -1,30 +1,32 @@
...
-before_script:
-  - composer self-update
-  - echo 'memory_limit = -1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - composer require symfony/symfony:${SYMFONY_VERSION} --prefer-source
+before_install:
+  - composer self-update || true
+  - sh -c 'if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi;'
+  - composer require --no-update symfony/symfony:${SYMFONY_VERSION}
+  - COMPOSER_ROOT_VERSION=dev-master composer update $COMPOSER_FLAGS --prefer-source --no-interaction
   - vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh
```
Comment (WouterJ) [inline .travis.yml]: I prefer to use things the correct way:

```yaml
before_install:
  - composer self-update || true
  - sh -c 'if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi;'
  - composer require --no-update symfony/symfony:${SYMFONY_VERSION}

install: COMPOSER_ROOT_VERSION=dev-master composer update $COMPOSER_FLAGS --prefer-source --no-interaction

before_script: vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh
```

Comment (lsmith77) [inline .travis.yml]: honestly I just wanted the PR to be rebased and the install part to work .. 
lets do all of the above things in a new PR, ok?

**Comment by dbu** [comment]
locally the slugifier did not work as expected. lets see if it gets better on travis.

**Pull request closed without merging**